specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/examples/ExpressRoutePortAuthorizationDelete.json

```diff
@@ -8,7 +8,11 @@
   },
   "responses": {
     "200": {},
-    "202": {},
+    "202": {
+      "headers": {
+        "Azure-AsyncOperation": "<azure-operation-result>"
```

Comment:
[https://github.com/Azure/azure-rest-api-specs/blob/d1fe957e0f2ff85ba9d3b4a2037e4db4041de0c4/specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/expressRoutePort.json#L660](https://github.com/Azure/azure-rest-api-specs/blob/d1fe957e0f2ff85ba9d3b4a2037e4db4041de0c4/specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/expressRoutePort.json#L660)

shows that the final state is given by Location header. 

1. Please confirm if both the headers are being sent ? 

You could also mention the headers in "responses" in the above provided link, so that it can show what headers are being sent in the 202 response.


2. Same comment for other files too.